### PR TITLE
Feature englishsite translations

### PR DESCRIPTION
**Pull request by vijivasanthan**

### CODE REVIEW

**Comment by vijivasanthan** [comment]
[8ATD1: Fix I18N (English-languages) on webapp](https://trello.com/c/w0Jsx7uT/1021-8atd1-fix-i18n-english-languages-on-webapp)

File: app/scripts/locals.js [hunk 1]
```diff
@@ -1017,8 +1020,8 @@ define(
           on: 'On',
           off: 'Off',
           useExternalId: {
-            title: 'Telefoonnummer van de beller',
-            info: 'Als deze optie is ingeschakeld en u wordt gebeld via TeamTelefoon, dan ziet u in het scherm van uw telefoon het telefoonnummer van de beller. Als deze instelling is uitgeschakeld, dan ziet u in het scherm het telefoonnummer van het team.'
+            title: 'Telephone number of the caller',
```
Comment (leonjoosse) [inline app/scripts/locals.js]: I'd suggest to change this to 'Caller number'. @ThomasAbbink , what do you think?

File: app/scripts/locals.js [hunk 2]
```diff
@@ -1017,8 +1020,8 @@ define(
...
+            info: 'If this option is enabled and you are called via Team Phone, you will see the caller telephone number in the screen of your telephone. If this setting is disabled, you will see the phone number of the team on the screen.'
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Please change "Team Phone" to "TeamTelefoon" or "TeamTelephone" (depends a bit on what Stefan likes as English product name)

File: app/scripts/locals.js [hunk 3]
```diff
@@ -681,7 +683,7 @@ define(
           order: 'Order',
           logs: 'Logs',
           scenario: 'Scenario',
-          nieuw: 'Nieuw',
+          new: 'Nieuw',
```
Comment (leonjoosse) [inline app/scripts/locals.js]: Looks like Dutch text in the English translation? Also, 'new' is already defined at line 670

**Pull request merged**